### DISCUSSION TITLE
how to write these repeated signs in Tex?

### DISCUSSION THREAD

**Discussion by goodgame365** [CoderLine/alphaTab#806] [General]
Hello
I don't know how to write these repeated signs in Tex,
 Can it be achieved at present，Thank,very much!

![160561474-84fe3dca-59a1-4580-910e-5db2cdfd5a74](https://user-images.githubusercontent.com/17884664/160647926-98098584-9dcb-49ed-8df6-e4e96f14968d.jpg)

**Comment by Danielku15** [comment]
Covered through https://github.com/CoderLine/alphaTab/discussions/671 and https://github.com/CoderLine/alphaTab/pull/791

**Comment by goodgame365** [comment]
thank verymuch
